app/models/user.rb

```diff
@@ -10,6 +10,26 @@ class User < ApplicationRecord
                     minimum: Settings.validations.length.digit_6},
            format: {with: VALID_EMAIL_REGEX}
   has_secure_password
+
+  class << self
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
+  end
+
+  def authenticated? remember_token
+    BCrypt::Password.new(remember_digest).is_password? remember_token
+  end
+
+  def forget
+    update_attribute :remember_digest, nil
```

Comment:
dùng `update_column`
```suggestion
    update_column :remember_digest, nil
```